src/core/store.ts

```diff
@@ -804,17 +786,19 @@ export const createStore = (
       })
       return
     }
-    const pending = Array.from(pendingMap)
-    pendingMap.clear()
-    pending.forEach(([atom, prevAtomState]) => {
-      const atomState = getAtomState(undefined, atom)
-      if (atomState && atomState.d !== prevAtomState?.d) {
-        mountDependencies(atom, atomState, prevAtomState?.d)
-      }
-      const mounted = mountedMap.get(atom)
-      mounted?.l.forEach((listener) => listener())
-    })
-    if (typeof process === 'object' && process.env.NODE_ENV !== 'production') {
+    while (pendingMap.size) {
```

Comment:
This looks like a good correctness improvement possibly - my state management framework has the same queue draining logic. But, is it needed for the build changes?

Reply:
Oooooooops, I was accidentally including my wip code in #950. That's why tests failed. Will revert them.